core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -25,6 +25,99 @@
   {{dependencies_html}}
 {% endblock header_js %}
 
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb ng-cloak" ng-controller="CollectionEditorNavbarBreadcrumb">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      <span ng-if="collection">
+        <[collection.getTitle()]>
+        <span class="oppia-navbar-breadcrumb-separator" style="margin-left: 5px;"></span>
+        <span>
+          <[getCurrentTabName()]>
+        </span>
+      </span>
+      <span ng-if="!collection">
+        Loading...
+      </span>
+    </li>
+  </ul>
+{% endblock navbar_breadcrumb %}
+
+{% block local_top_nav_options %}
+  <ul class="nav navbar-nav oppia-navbar-nav navbar-right ng-cloak" ng-controller="CollectionEditorNavigation" style="margin-left: 20px;">
```

Comment:
Yes, this should also be directive and `CollectionEditorNavbar` will be better name for it